executor/show.go

```diff
@@ -166,11 +166,28 @@ func (e *ShowExec) fetchShowEngines() error {
 	return nil
 }
 
+// let information_schema be the first database
+func replaceInformationFirst(dbs []string) {
```

Comment:
how about s/replaceInformationFirst/moveInfoSchemaToFront/?

Reply:
good name.

Reply:
Another question:
how to run just one file or one test case for the function,  like TestShowVisibility in the show_test.go to replace make dev for all test files.

Reply:
If the test resides in the xx directory:
```sh
cd xx
go test -check.f TestShowVisibility
```

Reply:
Thanks!